src/app/services/jira.service.ts

```diff
@@ -115,6 +115,9 @@ export class JiraService {
                     JobStatus.in_progress,
                     api_call,
                     {'tms_id': tms_id});
+                setTimeout(() => {
+                  new_job.set_title(`${new_job.get_title()} | This may take some time, feel free to close the window, we will email you when the projects are imported.`);
+                }, 30*1000);
```

Comment:
@ZachElkins , in another PR would be great to make it a well named constant so it is easier to reconfigure later. and probably use in the two places